ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -284,6 +289,25 @@ export const WorkspaceCard = fp.flow(withNavigation)(
                     style={{ justifyContent: 'flex-end', marginLeft: '1.2rem' }}
                   >
                     <FlexRow style={{ alignContent: 'space-between' }}>
+                      {accessTierShortName ===
```

Comment:
idea: is it worth splitting this out into a "workspace icons" component?

Reply:
I like it, i can make a PR for this